### DISCUSSION TITLE
Save results to db

### DISCUSSION THREAD

**Discussion by mbproducties** [grafana/grafana-infinity-datasource#412] [Q&A]
Is it possible to save a dashboard result in a database?

I have made a icecast http query for all listeners 
For bughunting i need to save these information

**Comment by yesoreyeram** [comment]
Nope. If you want to track single number from the response, [Grafana recorded queries](https://grafana.com/docs/grafana/latest/enterprise/recorded-queries/) is your option(enterprise feature). But again that is nothing to do with the plugin.
